.github/renovate.json

```diff
@@ -2,6 +2,8 @@
   "extends": [
     "config:base",
     ":separateMultipleMajorReleases",
+    ":separatePatchReleases",
+    ":widenPeerDependencies",
```

Comment:
Does this make sense if we don't have peer deps in this project?

Reply:
No it doesn't. I included it in the Modern Web Renovate config but I didn't think to check if this project needed it.

Reply:
Ok! Then please remove it! Thanks!